remote/remote_state_s3.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 const (
-	lockTableDeprecationMessage = "Remote state configuration 'lock_table' attribute is deprecated; use 'dynamodb_table' instead."
+	lockTableDeprecationMessage       = "Remote state configuration 'lock_table' attribute is deprecated; use 'dynamodb_table' instead."
+	s3BucketAccessLoggingTargetPrefix = "TFStatelogs"
```

Comment:
Is a trailing slash needed here?

Reply:
Also, NIT, but consider naming the var `defaultS3BucketAccessLoggingTargetPrefix`